src/app/configurations/allocations/calculations/calculations.controller.spec.js

```diff
@@ -41,6 +41,9 @@ describe('CalculationsController', function () {
           return $q.when([])
         })
       })
+      $provide.service('userStateService', function () {
```

Comment:
Just as a tip, I like to use `angular.noop` in these situations, e.g.: `$provide.service('userStateService', angular.noop)`